Makefile

```
@@ -42,6 +42,8 @@ test_63: builder_63
 	rm -f Cargo.lock
 	$(DOCKER_RUN) ${TAG_63} cargo test || true
 	$(DOCKER_RUN) ${TAG_63} cargo update -p home:0.5.9 --precise 0.5.5 || true
+	$(DOCKER_RUN) ${TAG_63} cargo update -p tokio-macros:2.4.0 --precise 2.3.0 || true
+	$(DOCKER_RUN) ${TAG_63} cargo update -p tokio:1.39.3 --precise 1.38.1 || true
```

This should be updated to match `cont_integration.yml`:
```suggestion
	$(DOCKER_RUN) ${TAG_63} cargo update -p tokio:1.39.3 --precise 1.38.1 || true
	$(DOCKER_RUN) ${TAG_63} cargo update -p cc --precise 1.0.105 || true
```